SteamKit2/SteamKit2/Util/Utils.cs

```diff
@@ -363,42 +365,95 @@ public static uint GetIPAddress( IPAddress ipAddr )
             return BitConverter.ToUInt32( addrBytes, 0 );
         }
 
-
-        public static uint EndianSwap( uint input )
+        public static IPAddress GetIPAddress( CMsgIPAddress ipAddr )

```

Comment:
Can we put this in `SteamKit2.Internal` alongside `CMsgIPAddress`? It seems odd having this function cross-namespace.

Also, IMO I think this makes more sense as an extension method.

Reply:
It won't be in `Internal` if protobufs update is merged.